doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3338,25 +3361,25 @@ Multiprotocol Service Layer (MPSL)
 
 The issues in this section are related to :ref:`nrfxlib:mpsl`.
```

Comment:
Please add a known issue in this section:

```
.. rst-class:: v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0

DRGN-22314: MPSL can encounter fatal errors and crashes when used without SoftDevice Controller or the 802.15.4 radio driver
  This happens because allocation functions in the :file:`nrfx_ppi.h` and :file:`nrfx_dppi.h` APIs can allocate channels reserved by MPSL. 
```